src/components/Header/Header.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { ChangeEvent } from 'react';
+
+type Props = {
+  inputValue: string;
+  newTodoField: React.RefObject<HTMLInputElement>;
+  isExist: boolean;
+  inputDisabled: boolean;
+  handleInputChange: (event: ChangeEvent<HTMLInputElement>) => Promise<void>;
+  handleAddTodo: (event: ChangeEvent<HTMLFormElement>) => Promise<void>;
+};
+
+export const Header: React.FC<Props> = ({
+  inputValue,
+  newTodoField,
+  isExist,
+  inputDisabled,
+  handleInputChange,
+  handleAddTodo,
+}) => {
```

Comment:
I don't see where you use this component. Please add and commit all files.

Reply:
Excuse me, I don't understand this comment, because there is this component in app.tsx and application can't work without it. Or is it something wrong?
![22](https://user-images.githubusercontent.com/110180778/202103722-f82141d5-0829-4ded-8923-8f10ae8e9ad8.jpg)